static/sass/_patterns_social.scss

```diff
@@ -0,0 +1,44 @@
+@mixin p-social {
+  .p-social__heading {
+    color: $color-mid-dark;
+    font-size: .875rem;
+    text-transform: uppercase;
+  }
+
+  .p-social__heading + * {
+    margin-top: .5rem;
```

Comment:
Default Vanilla typography adds a lot of margin (2rem) after the heading. So we need to override it so it doesn't look to far away from the icons...

Reply:
We should avoid using `*`. I think you should address this by something like:

``` sass
.p-social-list {
  @extends .p-inline-list;
  margin-top: 1rem;  // I think 0.5rem is a bit too close
}
```